styles/style.css

```diff
@@ -0,0 +1,358 @@
+  display: grid;
+  padding-left: 5rem;
+  padding-right: 6rem;
+  padding-top: 2rem;
+  padding-bottom: 6rem;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  grid-template-columns: 36% 1fr 30%;
+}
+header > div * {
+  margin: 0;
+}
+header img[alt="a white dog logo image for header"] {
+
+}
+header img[alt="a white color cart icon in the header"] {
+  width: 35px;
+  padding-right: 0.4rem;
+}
+header .cart-icon-money {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: baseline;
+}
+header p {
+  color: white;
+  font-size: 0.8rem;
+}
+header>div div:first-child {
+  grid-area: 1 / 1 / 5 / 2;
+}
+header div h1 {
+  color: #81B741;
+  font-size: 1.2rem;
+}
+header>div div:nth-child(2) {
+  grid-area: 1 / 3 / 2 / 4;
+}
+header>div button {
+  grid-area: 2 / 3 / 3 / 4;
+  background-color: #81B741;
+  border: none;
+  width: 5rem;
+  color: white;
+  width: 7rem;
+  height: 1.5rem;
+  justify-self: flex-end;
+}
+/* header style end */
+/* nav style start */
+header nav {
+  background-color: #EBEBEB;
+  padding-left: 9.5rem;
+}
+header nav ul li {
+  display: inline-flex;
+}
+header nav ul {
+  padding: 0;
+}
+header nav ul li:not(:last-child)::after {
+  content: '/';
+  padding: 0.9rem;
+}
+/* nav style end */
+/* footer style start */
+footer {
+  background-color: #037EAA;
+  padding: 2.5rem;
+}
+footer ul {
+
+  text-align: center;
+  padding-top: 1rem;
+  padding-bottom: 1.5rem;
+}
+
+footer img {
+  max-width: 30%;
+  display: block;
+  padding-bottom: 1.2rem;
+}
+footer ul li {
+  display: inline-flex;
+}
+footer ul li a {
+  color: white;
+}
+footer ul li:not(:last-child)::after {
+  content: '/';
+  color: white;
+  padding: 0.5rem;
+}
+footer p {
+  color: white;
+  text-align: center;
+}
+/* footer style end */
+/* dog-card style start */
+.container-dog-card {
+  border: 2px solid #EBEBEB;
+  padding-bottom: 0.7rem;
+}
+.button-dog-card {
+  background-color: #81B741;
+  color: white;
+  font-weight: bold;
+  width: 90%;
+  border-radius: 2px;
+  border: none;
+  padding: 0.4rem 3rem 0.4rem 3rem;
+  display: flex;
+}
+.name-dog-card {
+  padding: 0.3rem 5% 0.3rem 5%;
+  color: #103E67;
+  font-size: 0.95rem;
+}
+.description-dog-card {
+  padding: 0.3rem 5% 0.6rem 5%;
+}
+.label-cost-dog-card, .description-dog-card, .number-cost-dog-card {
+  font-size: 0.7rem;
+  color: black;
+  font-family: sans-serif;
+}
+.label-cost-dog-card {
+  font-weight: bold;
+  display: inline;
+  padding: 0.3rem 0 0.3rem 5%;
+}
+.number-cost-dog-card{
+  display: inline;
+}
+/* dog-card style end */
+/* home page style start */
+.home-page {
+  display: flex;
+  flex-direction: column;
+}
+.home-page div:first-child {
+  display: grid;
+  grid-template-columns: 40% 60%;
+  padding: 1.5rem 0 1.5rem 0;
+}
+.home-page >div:first-child h2 {
+  color: #103E67;
+  padding-bottom: 0.8rem;
+}
+
+.home-page div:first-child img {
+  padding-right: 2.5rem;
+}
+.home-page >div:nth-child(2) {
```

Comment:
rather than using nth-child to select these, instead give the div's a class and target them that way